var/spack/repos/builtin/packages/py-pytaridx/package.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyPytaridx(PythonPackage):
```

Comment:
Does this project have any relationship to https://pypi.org/project/taridx/?

Reply:
interesting. not that one, but this one is the same: https://pypi.org/project/pytaridx.